src/gui/runtime/plugin/nvim_gui_shim.vim

```diff
@@ -21,6 +21,14 @@ function! GuiClose() abort
   call rpcnotify(0, 'Gui', 'Close', v:exiting)
 endfunction
 
+" Open new window
+function! GuiNewWindow(...) abort
+	let server = get(a:, 0, 0)
+	let nvim = get(a:, 1, 0)
+
+	call rpcnotify(0, 'Gui', 'NewWindow', server, nvim)
+endfunction
```

Comment:
This is a function change. I've made the parameters optional.

Is that what you intended?

Before this, I was unable to call `:call GuiNewWindow()`.

Reply:
Yes, that was my intention. I just did that to get it working but then forgot to do it the right way. Thanks for fixing it.